### ISSUE TITLE
Layout Preview Speed

### ISSUE THREAD

**Issue by gotjoshua** [qgis/QGIS#51068]
### Feature description

![image](https://user-images.githubusercontent.com/8686454/204850036-e8ced7e4-71a8-4813-a2fd-01d21e2f70bf.png)

I recently discovered (via https://github.com/qgis/QGIS/issues/47033#issuecomment-1206114783) that my Layout view was so slow to render the preview because the export resolution was set too high.

I propose an additional field for preview resolution that defaults to half the screen resolution, and maxs at 2x screen resolution. Alternatively or additionally, it could be possible to dynamically calculate the preview resolution based on the current zoom and view window. However I think that decoupling the print output resolution from the preview resolution could already prevent slow down when preparing high res prints.

**Comment by weca-theo** [comment]
Yes, I think Export resolution setting should only apply to the final exported file, not to the QGIS preview screen.

**Comment by northcoastam** [comment]
I can confirm this is an issue for me using 3.28.5.  Most obvious for me with a large page size layout with an image layer, vrt or ecw, on a network share.  This is unrelated to #47033 and #48326 - the slow rendering occurs with only a Map containing a single layer - no Atlas or Legend.

Reducing the export resolution speeds up rendering.  Problem did not exist in QGIS 3.4.
